ironfish/src/serde/BlockTemplateSerde.ts

```diff
@@ -2,7 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-import { Block } from '../primitives/block'
+import { Block, RawBlock } from '../primitives/block'
```

Comment:
Are these changes intended to be in this PR?

Reply:
They are. The hashing code deals with `RawBlock` and `RawBlock` header a lot so its very helpful to be able to go from `SerializedBlockTemplate` to `RawBlock` and back. I can do it in another PR though